ui/src/app/admin/@tab/manage-person/page.tsx

```diff
@@ -0,0 +1,20 @@
+import { managePersonResults } from '@/lib/fetchers';
+import PersonTable from '@/app/admin/_components/personTable';
+import { memberAttributeResults } from '@/lib/actions';
+
+const PersonTab = async (searchParams) => {
```

Comment:
Make sure to set a type for searchParams, it would probably be: `searchParams: { searchUid: string }`. Also I recommend changing the searchParam name to just `uhIdentifier` because that naming would cover both uid and uhUuid.